modules/understanding-upgrade-channels.adoc

```diff
@@ -8,16 +8,43 @@
+* `eus-4.y` (only when running an even-numbered 4.y cluster release, like 4.10)
+
+If you do not want the Cluster Version Operator to fetch available updates from the upgrade recommendation service, you can use the `oc adm upgrade channel` command in the OpenShift CLI to configure an empty channel. This configuration can be helpful if, for example, a cluster has restricted network access and there is no local, reachable upgrade recommendation service.
+
+[WARNING]
+====
+Red Hat recommends upgrading to versions suggested by Openshift Update Service only. For minor version upgrade, versions must be contiguous. Red Hat does not test upgrades to noncontiguous versions and cannot guarantee compatibility with earlier versions.
+====
+
+endif::openshift-origin[]
+ifdef::openshift-origin[]
+{product-title} {product-version} offers the following upgrade channel:
+
+* `stable-4`
+
+endif::openshift-origin[]
 
 [id="candidate-version-channel_{context}"]
 == candidate-{product-version} channel
 
-The `candidate-{product-version}` channel contains candidate builds for a z-stream ({product-version}.z) and previous minor version releases. Release candidates contain all the features of the product but are not supported. Use release candidate versions to test feature acceptance and assist in qualifying the next version of {product-title}. A release candidate is any build that is available in the candidate channel, including ones that do not contain link:https://semver.org/spec/v2.0.0.html#spec-item-9[a pre-release version] such as `-rc` in their names. After a version is available in the candidate channel, it goes through more quality checks. If it meets the quality standard, it is promoted to the `fast-{product-version}` or `stable-{product-version}` channels. Because of this strategy, if a specific release is available in both the `candidate-{product-version}` channel and in the `fast-{product-version}` or `stable-{product-version}` channels, it is a Red Hat-supported version. The `candidate-{product-version}` channel can include release versions from which there are no recommended updates in any channel.
+The `candidate-{product-version}` channel contains pre-release channels with alpha, beta, and release candidate builds and versions. The `candidate` channel contains candidate builds for a z-stream and previous minor version releases.
+
+Release candidates (RC) contain all features of the product, but may not be supported. A release candidate is any build that is available in the candidate channel, including ones that do not contain link:https://semver.org/spec/v2.0.0.html#spec-item-9[a pre-release version] such as `-rc` in their names.
+
+The `candidate-{product-version}` channel can include release versions from which there are no recommended updates in any channel.
 
 You can use the `candidate-{product-version}` channel to upgrade from a previous minor version of {product-title}.
 
+Example channel name: `candidate-{product-version}`.
```

Comment:
I'm not sure what this line is doing.  We've been using `candidate-{product-version}` throughout the whole section.  Is it trying to underline the fact that this same description likely applies to `candidate-4.(y-1)`, etc., without asking folks to click through to look at the 4.(y-1) docs for a `candidate-4.(y-1)` description?